jjb/device/device-sdk-c.yaml

```diff
@@ -1,20 +1,28 @@
 ---
+
 - project:
     name: device-sdk-c
-    project-name: device-sdk-c
+    project-name:  device-sdk-c
     project: device-sdk-c
-    build-node: centos7-builder-2c-1g
-    nexus-group-id: device-sdk-c
     mvn-settings: device-sdk-c-settings
-    archive-artifacts: '$WORKSPACE/device-sdk-c/build/release/*.gz'
-    pre_build_script: ''
-    git_url: '$GIT_URL/$PROJECT'
-    build_script: '$WORKSPACE/device-sdk-c/scripts/build.sh'
-    build_dir: '$WORKSPACE/device-sdk-c/scripts/build'
+    docker_name: docker-device-sdk-c
+    docker_root: ''
+    docker_build_args: '-f scripts/Dockerfile.alpine-3.7'
+    archive-artifacts: '**/release/*.tar.gz'
+    post_build_script: !include-raw-escape: shell/device_sdk_c_build.sh
+    artifact-version: '0.1.0-SNAPSHOT'
```

Comment:
Not really sure how the `artifact-version` is used here. I think this was a copy/paste from another template.

Reply:
It was a copy/paste of the template based on device-grove-c. 
The build script is tagging the artifact with 1.0.0 so it may not be needed.

Reply:
Have you tested the job without this line? If it's not needed, it should be removed. And even if it is needed, you should probably make it something more appropriate (that is, a generic tag that indicates that no version was passed to the job).

Reply:
Not sure as to how I would test this on the sandbox. I assume artifact publishing would occur in the "merge" job and that is not allowed on the sandbox. I saw no reference to `artifact-version` in the `jjb/edgex-templates-docker.yaml` file. Not sure if it spider's up all the way up to the LF managed templates though. Hopefully you have knowledge in that area.